tests/geometry-calculation.test.js

```diff
@@ -0,0 +1,78 @@
+const Line = require('../modules/ecs6-class/line');
+const Point = require('../modules/ecs6-class/point');
+const { calculateDistance, calculateJunctionPoint, isPointOnLine } = require('../modules/geometry-calculation');
+const point1 = new Point({ x: 4, y: 3 });
+const point2 = new Point({ x: 3, y: 2 });
+const line1 = new Line({ point1, point2, slope: 2, n: 3 });
+
+const point3 = new Point({ x: 2, y: 2 });
+const point4 = new Point({ x: 2, y: 1 });
+const line2 = new Line({ point1: point3, point2: point4, slope: 2, n: 5 });
+const line3 = new Line({ point1: point3, point2: point4, slope: 4, n: 3 });
+
+const point5 = new Point({ x: 2, y: 2 });
+const point6 = new Point({ x: 1, y: 1 });
+const line4 = new Line({ point1: point5, point2: point6, slope: 1, n: 0 });
+const line5 = new Line({ point1: point5, point2: point4, slope: 1, n: 3 });
+
```

Comment:
Where is the mocks for the line class?

Reply:
don't create instances of objects in the head of the module, each test gets its own objects to test